src/pages/Lend/AlgoliaPagination.vue

```diff
@@ -1,13 +1,101 @@
 <template>
-	<ais-pagination :padding="padding" />
+	<ais-pagination :padding="padding">
+		<ul class="ais-Pagination-list"
+			slot-scope="{
+			currentRefinement,
+			nbPages,
+			pages,
+			isFirstPage,
+			isLastPage,
+			refine,
+			createURL
+			}"
+		>
+			<!-- previous page -->
+			<li
+				class="ais-Pagination-item ais-Pagination-item--previousPage"
+				:class="isFirstPage ? 'ais-Pagination-item--disabled' : ''">
+				<span
+					v-if="isFirstPage"
+					:href="createURL(currentRefinement - 1)"
+					@click.prevent="refine(currentRefinement - 1)"
```

Comment:
@foochris please remove the click events and hrefs from the disabled states and set the aria label to either "Previous (Inactive)" or "Next (Inactive)".

This would be for the Prev and Next disabled states.

Reply:
@mcstover PR updated.